app/components/show/controls_component.rb

```diff
@@ -40,29 +40,41 @@ def symphony_record?
       doc.catkey_id.present?
     end
 
-    delegate :admin_policy?, :agreement?, :item?, :embargoed?, to: :doc
+    delegate :admin_policy?, :agreement?, :item?, :collection?, :embargoed?, to: :doc
 
     private
 
     def manage_release
-      render ActionButton.new(url: item_manage_release_path(pid), label: 'Manage release')
+      render ActionButton.new(url: item_manage_release_path(pid), label: 'Manage release', open_modal: true)
+    end
+
+    def apply_apo_defaults
+      return unless item? || collection?
+
```

Comment:
I added a second guard against this button being shown for APOs (you'll note that the view template itself actually doesn't render this button at all for APOs)